posts/2024/2024_06_21_Inigo_week_4.rst

```diff
@@ -0,0 +1,53 @@
+Week 4 into GSoC 2024: Weight transfer experiments, hardships, and results!
```

Comment:
We'll need a paragraph break here. (Image from `blog.html`)

![image](https://github.com/dipy/dipy.org/assets/115863770/988c3ffb-72d5-4e1e-bffe-ee62a2ae8e49)

Reply:
Thanks for the comment @robinroy03, but I am not sure how to add it. Could you give me a hint, please?

Reply:
@itellaetxe your new commit fixes it. The first paragraph will be highlighted on the blog page.